packages/dates/src/apply-time-zone-offset.ts

```diff
@@ -5,10 +5,15 @@ export function applyTimeZoneOffset(
   timeZone1?: string,
   timeZone2?: string,
 ) {
+  const dateToCheckForDSTOffset = new Date(date);
+  dateToCheckForDSTOffset.setHours(2);
+  const needsDSTOffset = date.getTimezoneOffset() !== dateToCheckForDSTOffset.getTimezoneOffset();
+  const dstOffset = needsDSTOffset ? 60 : 0;
```

Comment:
DST offsets are not always 60 minutes.
 - Lord Howe Island [currently uses](https://www.zainrizvi.io/blog/falsehoods-programmers-believe-about-time-zones/) a 30-minute offset
 - England used a 2-hour offset in the past
 - Russia effectively [applied](https://en.wikipedia.org/wiki/Time_in_Russia) a 2-hour offset when they changed their DST rules in 2011

I'm OK with shipping this as is for now, but we should open an issue reminding us to circle back and handle such cases in the future.

Reply:
So I have looked into these timezones and others, and it seems this solution still works fine. I haven't been able to observe any issues. It also looks like as of 2014 Russia isn't participating in DST? 
>  Daylight saving time is not used in Russia (since 26 October 2014)

Reply:
I think with this I am ok moving forward with this solution, just need to figure out the tests.